python-main.js

```diff
@@ -916,6 +916,39 @@ function web_editor(config) {
         return fullHex;
     }
 
+    // Trap focus in modal and pass focus to first actionable element
+    function focusModal() {
+        document.querySelector('body > :not(.vex)').setAttribute('aria-hidden', true);
+        var dialog = document.querySelector('.modal-div');
+        var focusableEls = dialog.querySelectorAll('a[href]:not([disabled]), button:not([disabled]), textarea:not([disabled]), input[type="text"]:not([disabled]), input[type="radio"]:not([disabled]), input[type="checkbox"]:not([disabled]), select:not([disabled])');
+        $(focusableEls).each(function() {
+            $(this).attr('tabindex', '0');
+        });
+        dialog.focus();
+        dialog.onkeydown = function(event) {
+            if (event.which == 9) {
+                // if tab key is pressed
+                var focusedEl = document.activeElement;
+                var numberOfFocusableEls = focusableEls.length;
+                var focusedElIndex = Array.prototype.indexOf.call(focusableEls, focusedEl);
+
+                if (event.shiftKey) {
+                    // if focused on first item and user tabs back, go to the last focusable item
+                    if (focusedElIndex == 0) {
+                        focuseableEls.item(numberOfFocusableEls - 1).focus();
+                        event.preventDefault();
+                    }
```

Comment:
If I tab back from the first element in the load/save modal, the "Or browse for a file." link,  I get the following error:

![image](https://user-images.githubusercontent.com/29712657/68140411-ab09df80-ff23-11e9-8b55-3529a53657c6.png)

It's probably meant to be `focusableEls`?